packages/mobile/src/components/core/FlatList.tsx

```diff
@@ -20,9 +23,56 @@ const CollapsibleFlatList = ({
   ...other
 }: CollapsibleFlatListProps) => {
   const scrollPropsAndRef = useCollapsibleScene(sceneName)
-  return <Animated.FlatList {...other} {...scrollPropsAndRef} />
+  return (
+    <View>
+      <PullToRefresh />
+      <Animated.FlatList {...other} {...scrollPropsAndRef} />
+    </View>
+  )
 }
 
+const AnimatedFlatList = forwardRef<RNFlatList, FlatListProps>(
```

Comment:
Not a super big deal but if we pass a named function to `forwardRef` instead of an anon function the component will have a name in devtools. We do it in stems!

Reply:
good good